AzureMonitorAgent/ama_tst/modules/install/check_ama.py

```diff
@@ -0,0 +1,92 @@
+import re
+import requests
+from bs4 import BeautifulSoup
```

Comment:
I don't think bs4 works without installing it on the customer's VM - I tested it in python2 and python3, and both couldn't find the module. I asked Dmitry about it and he mentioned that we should try and stick with industry standard tooling, and it seems like this is mostly just for parsing the HTML file. Can you find another tool to parse the HTML file instead that doesn't require `pip install <module>` (or worst-case scenario, create simple infrastructure and parse the file manually)?

Reply:
I changed from bs4 to lxml. It can be used on Azure VM without installation. It should work with python 2.7 and after.